assets/template.html

```diff
@@ -167,6 +167,7 @@ <h1>@title@</h1>
                             <li class="menu-item menu-item-type-custom menu-item-object-custom "><a href="mailto:info@micronaut.io"><i class="fas fa-envelope"><span class="sr-only">Mail</span></i></a></li>
                             <li class="menu-item menu-item-type-custom menu-item-object-custom"><a href="https://twitter.com/micronautfw"><i class="fab fa-twitter"><span class="sr-only">Twitter</span></i></a></li>
                             <li class="menu-item menu-item-type-custom menu-item-object-custom"><a href="https://matrix.to/#/#micronautfw_questions:gitter.im"><i class="fab fa-gitter"><span class="sr-only">Gitter</span></i></a></li>
```

Comment:
delete gitter. Discord is replacing that. Please read https://micronaut.io/2023/07/28/micronaut-moves-to-discord/